packages/bitcore-node/src/models/baseTransaction.ts

```diff
@@ -36,6 +36,7 @@ export interface ITransaction {
   weight?: number;
   vsize?: number;
   receipt?: Array<ITransactionReceipt>;
+  vinScriptSig?: Array<string>;
```

Comment:
Should we put this field to coin model instead?

Reply:
ok kub